src/SwedbankPay/Api/Service/Request.php

```diff
@@ -211,6 +276,72 @@ public function send()
             throw new ClientException('PAYEE_ID not configured in INI file or environment variable.');
         }
 
+        // Retrieve parameters of operation
+        $paymentId = $this->getPaymentId();
+        $paymentOrderId = $this->getPaymentOrderId();
+        $operationRel = $this->getOperationRel();
+        if (($paymentId || $paymentOrderId) && $operationRel) {
+            // Fetch payment info
+            $responseBody = $this->getClient()->request(
+                'GET',
+                $paymentId ? $paymentId : $paymentOrderId,
+                $this->getRequestResource()
+            )->getResponseBody();
+
+            /** @var ResponseInterface $response */
+            $response = new Response($this, $responseBody);
+
+            // Check fields
+            if (in_array(
+                $this->getOperationRel(),
+                [
+                    'sales',
+                    'transactions',
+                    'verifications',
+                    'authorizations',
+                    'captures',
+                    'cancellations',
+                    'reversals',
+                    'urls',
+                    'payeeInfo',
+                    'metadata',
+                    'payer',
+                    'payments',
+                    'current_payment'
+                ]
+            )) {
+                // Configure request
+                $responseData = $response->getResponseData();
+                if (isset($responseData['payment'][$this->getOperationRel()])) {
+                    $this->setRequestEndpoint($responseData['payment'][$this->getOperationRel()]['id']);
+                } elseif (isset($responseData['payment_order'][$this->getOperationRel()])) {
+                    $this->setRequestEndpoint($responseData['payment_order'][$this->getOperationRel()]['id']);
+                }
+
+                throw new ClientException(sprintf('Field %s is unavailable', $this->getOperationRel()));
+            } elseif (!$response->getOperationByRel($this->getOperationRel())) {
+                throw new ClientException(sprintf('Operation %s is unavailable', $this->getOperationRel()));
+            }
+
+            // Apply operation options
+            $operation = $response->getOperationByRel($this->getOperationRel());
+
+            // Configure request
+            $this->setRequestMethod($operation['method']);
+            $this->setRequestEndpoint($operation['href']);
+
+            // Get rid of full url. There's should be an endpoint only.
+            if (filter_var($operation['href'], FILTER_VALIDATE_URL)) {
+                $parsed = parse_url($operation['href']);
```

Comment:
Codacy found an issue: [The use of function parse_url() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=6123563)